moped-editor/src/queries/project.js

```diff
@@ -741,6 +748,17 @@ export const PROJECT_UPDATE_SPONSOR = gql`
   }
 `;
 
+export const PROJECT_UPDATE_LEAD = gql`
+  mutation ProjectUpdateLead($projectId: Int!, $entityId: Int!) {
+    update_moped_project(
+      where: { project_id: { _eq: $projectId } }
+      _set: { project_lead_id: $entityId }
+    ) {
+      affected_rows
+    }
+  }
+`;
+
```

Comment:
we don't do this consistently, but i think it's preferable to use the `update_<table>_by_pk` mutation when updating a single record. it takes away some risk of a accidentally updating multiple records.

the Hasura docs have examples of how to structure the variables the pk mutations, in this case `update_moped_project_by_pk`:

https://hasura.io/docs/latest/mutations/postgres/update/#auto-generated-update-mutation-schema

Reply:
done! thanks for linking me to the docs :)